community/kernel/src/docs/dev/batchinsert.asciidoc

```diff
@@ -57,3 +76,12 @@ component=neo4j-kernel
 source=examples/BatchInsertDocTest.java
 tag=configFileInsert
 ----
+
+=== Importing into an existing database ===
+
+Although it's a less common use case, the batch inserter can also be used to import data into an existing database.
```

Comment:
You cannot have a GDS and a BatchInserter going on the same store at the same time. We could say that, but we have code to prevent that, is what I'm saying.